build/rewrite.go

```diff
@@ -288,7 +288,7 @@ func sortCallArgs(f *File) {
 		}
 		rule := callName(call)
 		if rule == "" {
-			return
+			rule = "<complex rule kind>"
```

Comment:
I'd also extract the names from DotExpr nodes (such as `foo.bar(...)`). This way we can have custom sorting rules for them in `tables.NamePriority` too.

Reply:
Done (updated callName()).